docs/admin/kubelet-tls-bootstrapping.md

```diff
@@ -17,11 +17,18 @@ and progress on the feature is being tracked as [feature #43](https://github.com
 
 ## kube-apiserver configuration
 
-You must provide a token file which specifies at least one "bootstrap token" assigned to a kubelet bootstrap-specific group.
+You must provide a token which specifies at least one "bootstrap token" assigned to a kubelet bootstrap-specific group.
 This group will later be used in the controller-manager configuration to scope approvals in the default approval
 controller. As this feature matures, you should ensure tokens are bound to a Role-Based Access Control (RBAC) policy which limits requests
 (using the bootstrap token) strictly to client requests related to certificate provisioning. With RBAC in place, scoping the tokens to a group allows for great flexibility (e.g. you could disable a particular bootstrap group's access when you are done provisioning the nodes).
 
+There are two methods by which tokens can be handled.
+
+1. [Bootstrap Tokens](/docs/admin/bootstrap-tokens.md)
+2. [Token authentication file](###token-authentication-file)
+
+Using bootstrap tokens is the preferred method as the token authentication file (csv) is difficult to manage especially in HA scenarios.
```

Comment:
Okay so I'll remove the preferred method wording and describe it as an alpha feature. How does that sound?